startup.sh

```diff
@@ -1,18 +1,10 @@
 #!/bin/bash -eux
 
-docker network create --driver bridge db_network
-docker run --network=db_network --name postgres_for_matching_service -d -p 5432:5432 postgres
-
 # wait for the database to come up
-MAX_RETRIES=15
```

Comment:
Since it uses docker-compose now to startup the database, I was not able to do `docker exec`. I have just replaced it with waiting for 10s.